### ISSUE TITLE
Error while submitting Extension to Magento

### ISSUE THREAD

**Issue by ankursachdeva11** [magento/magento2#11325]
We have extension released in Magento with 1.3.1 version.

Now we are trying to Upload new extension with version 1.3.2

This is our composer.json
{
  "name": "celigo/magento2-netsuite-connector",
  "description": "A Magento 2 module for Celigo Magento2 NetSuite Connector",
  "type": "magento2-module",
  "version": "1.3.2",
  "license": [

  ],
  "require": {
    "php": "~5.6.5|7.0.2|7.0.4|~7.0.6",
    "magento/framework": "100.1.*"
  },
  "autoload": {
    "files": [ "registration.php" ],
    "psr-4": {
      "Celigo\\Magento2NetSuiteConnector\\": ""
    }
  }
}

On Technical Review i got this error:

Errors during installation

Please visit the Magento DevDocs guide in order to learn how to install extensions and use CLI.

Environment: Operating System - Centos 7, PHP Version - 7.0.23, Magento Platform - 2.1.9 CE

Command: composer require celigo/magento2-netsuite-connector:1.3.2'
Installation issue: Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

Problem 1
- The requested package celigo/magento2-netsuite-connector 1.3.2 exists as celigo/magento2-netsuite-connector[0.0.1, 1.1.0, 1.2.0, 1.3.0, 1.3.1] but these are rejected by your constraint.

Installation failed, reverting ./composer.json to its original content.
[Pipeline] }
[Pipeline] // stage
[Pipeline] sh
[DockerOperationsPHP70Magento21CEVarnish] Running shell script
+ echo Failure
Failure   

Why they are trying to install via composer if it does not exist in the composer via command composer require celigo/magento2-netsuite-connector:1.3.2

Can anyone please help me for the extension submit.

**Comment by ankursachdeva11** [comment]
I have again submitted extension.

Now i got this issue

Command: composer require celigo/magento2-netsuite-connector:1.3.2'
Installation issue: Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 1 install, 0 updates, 0 removals
- Installing celigo/magento2-netsuite-connector (1.3.2): Downloading (connecting...)Downloading (0%) Downloading (55%)Downloading (100%)
Installation failed, reverting ./composer.json to its original content.


[UnexpectedValueException] 
The checksum verification of the file failed (downloaded from https://repo.magento.com/archives/celigo/magento2-netsuite-connector/celigo-magento2-netsuite-connector-1.3.2.0.zip) 


require [--dev] [--prefer-source] [--prefer-dist] [--no-progress] [--no-suggest] [--no-update] [--no-scripts] [--update-no-dev] [--update-with-dependencies] [--ignore-platform-reqs] [--prefer-stable] [--prefer-lowest] [--sort-packages] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--] []...

[Pipeline] }
[Pipeline] // stage
[Pipeline] sh
[DockerOperationsPHP70Magento21CEVarnish] Running shell script
+ echo Failure
Failure

**Comment by logeecom** [comment]
Hi @ankursachdeva11, what did you do to get pass the first error? I got the same report. Thanks!

**Comment by ankursachdeva11** [comment]
@logeecom 
got reply from Magento Marketplace on this issue:

Yes, we have identified a bug with our install automated test and resolved the issue this morning. We are going to rerun the test today (I added your extension to this list). You will see a separate email either pass or fail (if for some valid reason). So no action is required on your part. Just be on the lookout for the email today.
